static/favicons/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Campaigns Library",
+    "short_name": "Campaigns Library",
+    "icons": [
+        {
+            "src": "/android-chrome-192x192.png",
+            "sizes": "192x192",
+            "type": "image/png"
+        },
+        {
+            "src": "/android-chrome-384x384.png",
+            "sizes": "384x384",
+            "type": "image/png"
+        }
+    ],
+    "theme_color": "#ffffff",
+    "background_color": "#ffffff",
+    "display": "standalone"
+}
```

Comment:
this looks like a pwa manifest...but a slightly incomplete and and stored in favicons. What's this for?

Reply:
I used https://realfavicongenerator.net/ to generate this icon set. i believe this is used for saving to the home screen on android. so probably the same as a pwa manifest yes. Real favicon generator walks you through a process where you can customise the icon presentation on a few different devices/contexts.

Reply:
ah okay i think this is the slimmed down version, so not a PWA but may set the bookmark icon?